script/verification/DisputeGameUpgrade.s.sol

```diff
@@ -86,4 +94,60 @@ abstract contract DisputeGameUpgrade is VerificationBase, SuperchainRegistry {
 
         require(address(faultDisputeGame.weth()) != address(permissionedDisputeGame.weth()), "weth-200");
     }
+
+    function precheckDisputeGames() internal view {
```

Comment:
(nit) It probably makes sense to rename `checkDisputeGameUpgrade` to `postcheckDisputeGames` for clarity.

Reply:
It also looks to me like most of the post-checks could be pre-checks?  If so, would be nice to clean that up, but can be for a future PR.

Reply:
Yes quite a few checks could be done as pre-checks instead. But it doesn't matter too much because all is happening during simulations first. This is how most FP upgrade tasks did it in the past too.

Worth fixing in a follow-up. Would like to get this one in as is.

Reply:
I decided to keep the name `checkDisputeGameUpgrade` because it says "check ... upgrade" which implies that it runs checks against an upgrade that has happened. I also want to keep the convention of naming the function `check...` for the function that should be run from the inheriting task contract.